resources/assets/js/components/UserEditForm.vue

```diff
@@ -219,8 +224,8 @@ export default {
       personal_email: { email, notGTEmail },
       phone: { maxLength: maxLength(15) },
       preferred_first_name: { alpha },
-      shirt_size: {},
-      polo_size: {},
+      shirt_size: { required },
```

Comment:
Technically we don't need your shirt size unless you pay dues, so I'm not sure if this validation is per se correct.